fix: visual inconsistency in app_settings

Sudhanva-Nadiger opened the pull request:
- fix: bottom padding for rate us tile
- fix: misalignment of share icon refactor: added user_preferences list tile
- which is used for both rate us and share app tile
- fix: The rounded corners of the country and the language pickers are different

<!-- 
Please name your pull request following this scheme: `type: What you did` this allows us to automatically generate the changelog
Following `type`s are allowed:
  - `feat`, for Features
  - `fix`, for Bug Fixes
  - `docs`, for Documentation
  - `ci`, for Automation
  - `refactor`, for code Refactoring
  - `chore`, for Miscellaneous things
-->
### Screenshot
<!-- Insert a screenshot to provide visual record of your changes, if visible -->

### Fixes bug(s)
<!-- change by appropriate issues. -->
<!-- Please use a linking keyword https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword -->
- Fixes: <!-- #1 Note: you can also use Closes: -->

### Part of 
- #525 
- #3810 <!-- Add the most granular issue possible -->

Sudhanva-Nadiger committed:
fix: visual inconsistency in app_settings

- fix: bottom padding for rate us tile
- fix: misalignment of share icon
refactor: added user_preferences list tile
- which is used for both rate us and share app tile

Sudhanva-Nadiger committed:
fix: The rounded corners of the country and the
language pickers are different

- added the same border radius to language picker

Sudhanva-Nadiger committed:
fix: add rounded corner to language picker

g123k commented on packages/smooth_app/lib/generic_lib/widgets/language_selector.dart:
```diff
@@ -132,6 +125,22 @@ class LanguageSelectorSettings extends StatelessWidget {
+      borderRadius: ANGULAR_BORDER_RADIUS,
+      child: Container(
+        decoration: const BoxDecoration(
+          borderRadius: BorderRadius.all(Radius.circular(100)),
```
Where does this 100 came from?
It should be shared with the other button

Sudhanva-Nadiger replied:
yeah, I just wanted to show it has no effect on the border because we are already using border-radius on inkwell 😄, but the same goes with the country picker, it used a container as well! it was wrapped with a container having borderradius as 10

Sudhanva-Nadiger replied:
should i remove it in both the places ?

g123k commented on packages/smooth_app/lib/pages/preferences/user_preferences_widgets.dart:
```diff
@@ -243,3 +243,37 @@ class UserPreferencesTitle extends StatelessWidget {
+      required this.leading,
+      required this.onTap,
+      required this.showDivider,
+      super.key});
```
Could you add an extra comma here, please?

Sudhanva-Nadiger replied:
yesssss

Sudhanva-Nadiger committed:
formatting

Sudhanva-Nadiger commented:
@g123k added the changes! Thanks

g123k commented on packages/smooth_app/lib/generic_lib/widgets/language_selector.dart:
```diff
@@ -132,6 +125,24 @@ class LanguageSelectorSettings extends StatelessWidget {
         );
       },
+      borderRadius: ANGULAR_BORDER_RADIUS,
+      child: Container(
```
You can remove the Container here, it's totally useless

Sudhanva-Nadiger replied:
Yesss I will make the suggested changes!

g123k reviewed (commented):
A small change for the Language selector +
Could you make the icon of the "Native App Settings" to the left, no icons are on the right in this screen

Sudhanva-Nadiger committed:
fix: inconsistency

- remove unnecessary container in language_selector
- inAppsettings tile made the icon leading instead of trailing

Sudhanva-Nadiger commented:
@g123k I have made the chnges ! sorry for the delay !

Sudhanva-Nadiger commented:
@g123k I have made the chnges ! sorry for the delay !

g123k reviewed (approved):
Looks better now, thanks for your changes!
